controllers/articles.controller.js

```diff
@@ -54,4 +54,29 @@ exports.postCommentByArticleID = (req, res, next) => {
       res.status(201).send({ comment });
     })
     .catch(next)
+};
+
+exports.patchArticleVotes = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+
+  if (isNaN(article_id)) {
+    return res
+      .status(400)
+      .send({ msg: "Bad Request: article_id must be a number" });
+  }
```

Comment:
Rather than doing this, PSQL will actually throw an error for you if the article_id is not a number, so your error handling middleware should be able to pick it up if you include the relevant code 👍